packages/@lwc/integration-karma/test-hydration/svg/with-empty-class/index.spec.js

```diff
@@ -1,7 +1,7 @@
 export default {
     test(elm, snapshots, consoleCalls) {
         // TODO [#3548]: inconsistent rendering of empty class/style
-        expect(elm.shadowRoot.querySelector('svg circle').getAttribute('class')).toEqual('');
+        expect(elm.shadowRoot.querySelector('svg circle').getAttribute('class')).toEqual(null);
```

Comment:
```suggestion
        expect(elm.shadowRoot.querySelector('svg circle').hasAttribute('class')).toBe(false);
```

I find it more readable personally